src/ast/document-descriptor.ts

```diff
@@ -146,6 +148,9 @@ export class Document implements Feature {
   getByKind(kind: 'import'): Set<Import>;
   getByKind(kind: string): Set<Feature>;
   getByKind(kind: string): Set<Feature> {
+    if (this._featuresByKind) {
```

Comment:
Do you need to push changes? Still checking for the existence of _featuresByKind